src/stretchy.js

```diff
@@ -254,11 +254,13 @@ const encloseSpan = function(inner, label, pad, options) {
     let img;
     const totalHeight = inner.height + inner.depth + 2 * pad;
 
-    if (label === "fbox") {
+    if (/(fbox)|(color)/.test(label)) {
         img = buildCommon.makeSpan(["stretchy", label], [], options);
-        if (options.color) {
+
+        if (label === "fbox" && options.color) {
             img.style.borderColor = options.getColor();
```

Comment:
Why do we use `options` for the color whereas in `buildHTML.js` we use `borderColor` and `bkgrndColor` directly from the group?

Reply:
The `fbox` border takes the current text color. The `options.getColor()` will do that successfully even if the color was defined using the KA one argument custom colors.  See PR #792.

I don't think we want to support behavior that for `\colorbox` and `\fcolorbox`. I think these two proposed functions should get only the colors specified in the arguments to the function